docs/compatibility.md

```diff
@@ -892,8 +894,7 @@ Casting from string to timestamp currently has the following limitations.
 | `"tomorrow"`                                                        | Yes               |
 | `"yesterday"`                                                       | Yes               |
 
-- <a name="Footnote1"></a>[1] The timestamp portion must have 6 digits for milliseconds.
- Only timezone 'Z' (UTC) is supported. Casting unsupported formats will result in null values.
```

Comment:
Why was the note about only supporting timezone `Z` dropped?

Reply:
If `Z` is required, the input string has to have the suffix `Z` otherwise the output will be null. Maybe this doc was already inaccurate? So should it be this?
```
If a timezone is provided then only timezone 'Z' (UTC) is supported.
```

Reply:
I meant, currently this doc (and the code) require that the input string must have a suffix `Z`. We should support input strings not having suffix `Z`.